packages/scss/src/components/_actionIcon.scss

```diff
@@ -0,0 +1,20 @@
+.actionIcon {
+	background: transparent;
+	border: 0;
+	border-radius: 50%;
+	color: _color("text.default");
+	cursor: pointer;
+	height: 2rem;
```

Comment:
Hum... I feel like it could lead to unwanted size modifications if the button is linked to his parent size. So I'd rather add a size option to keep the component encapsuled.